media/js/mozorg/home/statement.js

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+ (function($, Mozilla) {
+    'use strict';
+
+    var $promo = $('#promo-statement');
+    var cookieDurationDays = 2;
+    var cookiesOK = typeof Mozilla.Cookies !== 'undefined' && Mozilla.Cookies.enabled();
+    var storageKey = 'statement-04-2018';
+    var wasClosed = false;
+    var evtNamespace = 'promo-statement';
+    var $doc = $(window.document);
+
+    // see if visitor previously dismissed the banner
+    if (cookiesOK) {
+        wasClosed = Mozilla.Cookies.getItem(storageKey);
+    }
+
+    if (!wasClosed) {
```

Comment:
I'm not sure where is best, but should probably have some kind of viewport width check to make sure the text fits horizontally?

Reply:
All the fancy stuff should be tablet and up now.

Reply:
All the fancy stuff is desktop and up now.